### ISSUE TITLE
RPG Chat: Privacy Options (Toggle Read Reciepts On/Off)

### ISSUE THREAD

**Issue by ChaoticMarin** [RolePlayGateway/roleplaygateway#236]
Yo! I'd like to bring up some privacy concerns regarding the RPG Chat App that may not have been considered because of their ubiquity.

Namely, read receipts. I'm convinced read receipts were invented PURELY to make people angry and frustrated. In my eyes, they serve no purpose other than to mislead people about when someone is ready to respond to a message. Awful feature, has almost no positive benefits in practice as far as I can tell.

More importantly, though, read receipts are an insidious invasion of privacy. Personally, I never consented to anyone knowing when I'm "at" my computer or when I supposedly "saw" a message. Nor is there any option in the settings to disable them. You can have the app stop showing other people's read receipts, but you can't stop your app from sending them. We should have such an option! I can't tell you how many times someone has flipped out and blown up my DMs because they thought I was ignoring them when in fact I simply wasn't ready to respond yet due to something else going on in my life.

On a less frustrating level, disabling "You are typing" messages would also be a handy privacy option. I rarely run into trouble caused by those in comparison, but it's still information other users aren't entitled to that's being taken without consent.

**Comment by martindale** [comment]
Can you check the user settings / preferences for a "Send Read Receipts" option?

**Comment by ChaoticMarin** [comment]
Nope! There's no such option in my settings / preferences. Though now that I look, I realize there IS a "Send typing notifications" options which solves my other concern. I only went searching for the read receipts option before I posted the issue. Is that to say that this option will be included in a newer update?

![image](https://user-images.githubusercontent.com/6273506/73685973-21d68600-4695-11ea-9842-d2b59989ab21.png)
